dash.c

```diff
@@ -234,7 +234,7 @@ int dash_pipe(char **args)
 char *get_hist_file_path()
 {
 	static char file_path[128];
-	strcat(strcpy(file_path, getenv("HOME")), "/.dash_history");
+	strcat(strncpy(file_path, getenv("HOME"), 113), "/.dash_history");
```

Comment:
Is 113 just arbitrary?

Reply:
I used 113 because `file_path` has 128, and `/.dash_history` is14 characters, and I included one more for the terminating null character. 128 - 15 = 113.

Reply:
Got it, just read this again and it makes sense. I'm still not sure what lead me to set `file_path` to 128 in the first place, it's been quite a while since I looked back on this.